js/parts/ColumnSeries.js

```diff
@@ -609,8 +665,11 @@ seriesType('column', 'line',
             (groupPadding +
                 colIndex * pointOffsetWidth -
                 (categoryWidth / 2)) * (reverseStacks ? -1 : 1);
-        if (chart.maxColumnCount &&
-            series.options.ignoreNulls === 'centered') {
+        // Handle pointXOffset when series.ignoreNulls: 'normal'
+        // so the points are aligned to the center of category.
+        if (chart.maxColumnCount && ignoreNulls &&
+            ignoreNulls !== 'evenlySpaced' &&
+            ignoreNulls !== 'fillSpace') {
```

Comment:
Consider to use `ignoreNulls === 'center'` instead of two negations

Reply:
The purpose of this is to allow this implementation to work not only for `ignoreNulls: 'normal'`, but also for `ignoreNulls: true` and similar.
If you think it's no needed and it's not a good idea, then, please, let me know and I'll change it.